API.md

```diff
@@ -267,10 +267,10 @@ These methods are all called off of existing parsers, not from the `Parsimmon` o
 Returns a new parser which tries `parser`, and if it fails uses `otherParser`. Example:
 
 ```javascript
-var numberPrefix =
+var maybePlus =
```

Comment:
This can be plus or minus, so I think `prefix` is better here. Or maybe `maybePlusMinus`?